### ISSUE TITLE
[Upload Progress Monitoring Epic #3533] VolumeSnapshotter new plugin APIs for upload progress monitoring

### ISSUE THREAD

**Issue by dsu-igeek** [vmware-tanzu/velero#3753]
Implement the new VolumeSnapshotter APIs needed for upload progress monitoring as described in the upload-progress.md design doc.  Part of Epic #3533 Depends on #3285

**Comment by dsu-igeek** [comment]
No longer needed
